.github/workflows/release.yml

```diff
@@ -1,6 +1,8 @@
 name: Release
 on:
   push:
+    tags:
+    - '*'
```

Comment:
What about restricting to 'v*' for release tags?

Reply:
I checked the output of `git tag`. They all are release tags, so maybe it should be fine.